python/setup.py

```diff
@@ -214,37 +233,41 @@ def run(self):
                     dst_file = path.join(mac_os_static_libs_dir, lib_file_name)
                     shutil.copyfile(src_file, dst_file)
 
-            if not path.exists(path.join(tpie_lib_dir, 'libtpie.a')):
-                try:
-                    cpu_count = multiprocessing.cpu_count()
-                except:
-                    cpu_count = 1
+            CMAKE_CXX_FLAGS = '-fPIC -std=c++11'
+            if sys.platform == 'darwin':
+                CMAKE_CXX_FLAGS += ' -mmacosx-version-min=10.9'
 
-                CMAKE_CXX_FLAGS = '-fPIC -std=c++11'
-                if sys.platform == 'darwin':
-                    CMAKE_CXX_FLAGS += ' -mmacosx-version-min=10.9'
+            keyvi_build_cmd = 'mkdir -p {}'.format(keyvi_build_dir)
+            keyvi_build_cmd += ' && cd {}'.format(keyvi_build_dir)
+            keyvi_build_cmd += ' && cmake -D CMAKE_BUILD_TYPE:STRING=bindings ' \
```

Comment:
As `CMAKE_BUILD_TYPE` is usually used to specify build type, e.g. Release, Debug and etc. I'd suggest using `cmake --build` command with `--target` option. Also we will not need the IF-ELSE clause and TPIE workaround in CMakeLists.txt file. 

I was facing the same issue for Rust: https://github.com/narekgharibyan/keyvi-1/commit/385d61de9939e4ff7dd248ee5d4bfa93f66d753f

Reply:
@narekgharibyan 

the best alternative I can come up with:

```
add_custom_target(bindings
    COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:tpie> ${CMAKE_BINARY_DIR}
    COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:tiny-process-library> ${CMAKE_BINARY_DIR}
    DEPENDS tpie tiny-process-library
)
```

that create a target bindings (`make bindings`) and copies both libraries into the cmake build root. Otherwise it is hard to get the libs and a `make install` always triggers building everything.

Reply:
I see, but probably this is a better alternative.